lib/methods_for_similarity.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+
+def IOU(lista1, lista2):
+  # metoda przyjmuje jako argument dwie listy i zwraca ich indeks Jaccarda, czyli liczbę z przedzialu [0,1]
+  intersection = np.intersect1d(lista1, lista2)
```

Comment:
numpy sugeruje, że tego typu będziecie właście używać w części koncowej projektu. To jest już jakaś zespołowa decyzja o której nie wiem?

Reply:
miało chyba być dla list pythonowych nie numpy.

Reply:
Zrobione. Metoda nie używa numpy